app/build.gradle.kts

```diff
@@ -1,19 +1,22 @@
 plugins {
     id("com.android.application")
+    kotlin("android")
 }
 
 android {
-    compileSdk = 33
-    buildToolsVersion = "30.0.3"
+    compileSdk = 34
 
     defaultConfig {
         applicationId = "org.asteroidos.sync"
         minSdk = 24
-        targetSdk = 33
+        targetSdk = 34
         versionCode = 29
         versionName = "0.29"
         ndk.abiFilters.clear()
         ndk.abiFilters.add("arm64-v8a")
+        ndk.abiFilters.add("armeabi-v7a")
+        ndk.abiFilters.add("x86")
+        ndk.abiFilters.add("x86_64")
```

Comment:
Uh, now these are re-added and cgi is deleted. That's a lot of back and forths to review. I realize the PR is marked as draft, it would be nice to clean up this backlog. I'll stop reviewing here until then ;)